src/coreclr/src/jit/CMakeLists.txt

```diff
@@ -402,11 +404,12 @@ endfunction()
 set(JIT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
 if (FEATURE_MERGE_JIT_AND_ENGINE)
-  # Despite the directory being named "dll", it creates a static library "clrjit_static" to link into the VM.
-  add_subdirectory(dll)
   add_subdirectory(crossgen)
 endif (FEATURE_MERGE_JIT_AND_ENGINE)
 
+# Despite the directory being named "dll", it creates a static library "clrjit_static" to link into the VM.
```

Comment:
Can we give it a better name, like `static`, so that this comment is not necessary?

Reply:
Yes, i think so. And it might be a good time to rename this,